spec/plugman/util/metadata.spec.js

```diff
@@ -32,6 +32,7 @@ const fsMock = {
 };
 
 // expect fsMock to only operate on fetchJsonPath
+// eslint-disable-next-line
 Object.entries(fsMock).map(([key, fn]) => {
```

Comment:
Would `forEach` work without a warning?
```js
Object.entries(fsMock).forEach(([key, fn]) => {
```

Reply:
I just tested with `forEach` and it worked OK.

The warning was about how:
> Array.prototype.map() expects a return value from arrow function

Reply:
Yah, `map` is for creating a new list out of another list... so if the goal was simply iterate then `forEach` is definitely what should have been used.